app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
Right, you got me.  In theory this should be enough: https://github.com/katello/katello/commit/be3e8de#diff-84f0b46367dabf5e0710a0fe3aca7627R143

But it's not. I don't particularly understand why the Hostgroup Jail makes it work.  But we have it in the Host::Managed jail, and that seems to be the only reason it renders correctly for host provisioning.